lib/solr_wrapper/instance.rb

```diff
@@ -222,17 +226,23 @@ def with_collection(options = {})
     end
 
     ##
-    # Clean up any files solr_wrapper may have downloaded
-    def clean!
+    # Clean up any files solr_wrapper may have downloaded, except the solr-{ver}.zip file
+    def clean
```

Comment:
OK, pushing up something with `#clean! clean_downloads: false`, so the default action is not to clean up the downloads.  The commit also focuses on removing files only (.md5, .zip download files) and not the entire download directory (in case anyone sets .solr_wrapper with a download_dir like `~/Downloads`).  The more likely scenario is that multiple projects use different versions of solr and share a download_dir, so any instance of solr_wrapper should clean up only the files configured for this instance (or project).